tests/Canvas/CPDFTest.php

```diff
@@ -129,6 +129,7 @@ public static function fontSupportsCharProvider(): array
     /**
      * @dataProvider fontSupportsCharProvider
      */
+     #[\PHPUnit\Framework\Attributes\DataProvider('fontSupportsCharProvider')]
```

Comment:
I think importing the attribute classes is nicer in general.

Reply:
Oh, just realized that this wouldn’t work, as they are unavailable before PHP 8. Never mind.